src/client/Microsoft.Identity.Client/Platforms/Android/AndroidPlatformProxy.cs

```diff
@@ -217,5 +218,10 @@ private static bool IsChromeEnabled()
         }
 
         public override bool UseEmbeddedWebViewDefault => false;
+
+        public override IDeviceAuthManager CreateDeviceAuthManager()
+        {
+            return new AndroidDeviceAuthManager();
```

Comment:
There is no need to create dummy device managers for each platform. Just create a `NullDeviceAuthManager` for all platforms that doesn't support this.